README.md

```diff
@@ -189,4 +191,4 @@ The output fields for each line are:
 
 ## Related project
 - [Gocyclo](https://github.com/fzipp/gocyclo) where the code are based on.
-- [Cognitive Complexity: A new way of measuring understandability](https://www.sonarsource.com/docs/CognitiveComplexity.pdf) white paper by G. Ann Campbell.
\ No newline at end of file
+- [Cognitive Complexity: A new way of measuring understandability](https://www.sonarsource.com/docs/CognitiveComplexity.pdf) white paper by G. Ann Campbell.
```

Comment:
I did not change this line! do you have windows? maybe new line symbol is different. or some other invisible symbol.

Reply:
No I don't have it.